editors/intellij/gradle.properties

```diff
@@ -4,7 +4,7 @@ pluginGroup = com.github.biomejs.intellijbiome
 pluginName = Biome
 pluginRepositoryUrl = https://github.com/biomejs/biome
 # SemVer format -> https://semver.org
-pluginVersion = 0.0.2
+pluginVersion = 0.0.3
```

Comment:
This might not be needed since you did not change the plugin, only the workflow.